### ISSUE TITLE
Support formatting tables on templates

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#371]

**Comment by yaelberger-commits** [comment]
I'm curious about this one. There are ways to format tables in markdown. Is it an issue of the learning curve to figure out how that is the pain point for them?

**Comment by adriannelee** [comment]
Yes .. Notify doesn't support tables with Markdown because of the security and the accessibility of the output. But we haven't really learned what they were trying to do with a table.

**Comment by schancase** [comment]
this should probably be a feature or an epic
